### ISSUE TITLE
W4-Submission

### ISSUE THREAD

**Issue by Nedu** [Nedu/TwitterDemo#1]
My app is complete, please review. /cc @codepathreview

**Comment by avo1** [comment]
Hi @Nedu ,

It looks like you are missing some features from the required user stories. We will need to have this completed and updated within **24hr** from the deadline in order for you to receive credit for the assignment.

If you need help with the these features, please reach out to your professional mentors asu slack channel or post to universitysupport@codepath.com

Dave
CodePath

**Comment by avo1** [comment]
Hi @Nedu , it seems like you've updated something. However, I didn't see those functions either in your gif and/or your code. Please push your latest code and update your gif asap.

Reply to this issue when you're done.

Thanks,
Dave

**Comment by Nedu** [comment]
Hi, my gif and code is updated already and i can see it on my end

**Comment by avo1** [comment]
Hi, I cloned your project and run but I can only login. Nothing shows up. Are you sure this is your latest code? I don't even see any `tableView` in the `TwitterViewController.swift`

<img width="467" alt="screen shot 2016-02-24 at 7 05 27 pm" src="https://cloud.githubusercontent.com/assets/10482065/13284866/b8888ebc-db29-11e5-811f-f0743d2beba8.png">

**Comment by Nedu** [comment]
No, that isn't my latest code but i think I have figured out what went wrong with the help of my student coordinator. It seems that when I tried to push my latest code, i mistakenly overrode my new code with the code already on github. I'm working with them to restore that.

**Comment by Nedu** [comment]
Hi, I think i got it fixed now. Can you see my updated code and gif now?

**Comment by avo1** [comment]
Thanks for your hard work fixing it :+1: Nice work! The point of this homework was to explore an example of a full MVC application with a RESTful API. 
- You should be familiar with autoLayout now. The content of the tweet should be multiple lines.
- Good job implementing the fav/retweet number update with nice button images. However, you haven't implemented the API call for that function. You need to call the API for [Retweeting](https://dev.twitter.com/rest/reference/post/statuses/retweet/%3Aid)  and [favoriting](https://dev.twitter.com/rest/reference/post/favorites/create) to actually perform those actions. This task is quite hard and we decided to make it optional. So you still get credit for this assignment.

We have a detailed [Project 4 Feedback Guide](http://courses.codepath.com/snippets/ios_university/feedback_guides/project_4_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at universitysupport@codepath.com.

Dave
Codepath
